relationshipTypes/set.yml

```diff
@@ -17,6 +17,14 @@ roleTypes:
     label: "Groups that include {target}"
 links:
   - '@type': LinkType
+    id: linkTypes/subgroup>parent-group
+    relationshipType: relationshipTypes/set
     source: subgroup
     target: parent-group
     label: "are part of {target}"
+  - '@type': LinkType
+    id: linkTypes/parent-group>subgroup
+    relationshipType: relationshipTypes/set
+    source: parent-group
+    target: subgroup
+    label: "are part of {target}"
```

Comment:
That's right. 

This has evolved since then in my local. Since I'm developing this in tandem with the ```query``` branch I'll close this and do a pr when both are ready.